qcfractal/interface/collections/gridoptimization_dataset.py

```diff
@@ -11,7 +11,7 @@
 from .collection_utils import register_collection
 
 
-class GORecord(BaseModel):
+class GOEntry(BaseModel):
```

Comment:
Why the change from `Record` to `Entry` in all of these?

Reply:
A `Record` is specifically the computation record. These "Entries" are the starting inputs for records. We overloaded the term, this is removing that issue.